specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/definitions.json

```diff
@@ -1382,6 +1382,16 @@
       },
       "description": "Authorization login response contract."
     },
+    "AuthorizationConfirmConsentCodeRequestContract": {
+      "type": "object",
+      "properties": {
+        "consentCode": {
+          "type": "string",
+          "description": "The consent code from the authorization server after authorizing and consenting."
+        }
+      },
+      "description": "Authorization confirm consent code request contract."
+    },
     "AuthorizationProviderCollection": {
```

Comment:
>AuthorizationProviderCollection [](http://example.com/codeflow?start=5&length=31)

No need to return count in paginated queries. Test for structure of the collection responses for confirmation.

Reply:
Removed count property